README.md

```diff
@@ -69,13 +69,17 @@ This is the parameterization of the BAM stat tool, a copy is present in this rep
 Run it using the `dockstore` CLI:
 
 ```
-Usage:
-# fetch CWL
-$> dockstore tool cwl --entry quay.io/collaboratory/dockstore-tool-bamstats:1.25-3 > Dockstore.cwl
-# make a runtime JSON template and edit it (or use the content of sample_configs.json above)
-$> dockstore tool convert cwl2json --cwl Dockstore.cwl > Dockstore.json
-# run it locally with the Dockstore CLI
-$> dockstore tool launch --entry quay.io/collaboratory/dockstore-tool-bamstats:1.25-3 --json Dockstore.json
+# Fetch CWL workflow
+dockstore tool cwl --entry quay.io/collaboratory/dockstore-tool-bamstats:1.25-7 > Dockstore.cwl
```

Comment:
I tried this and I think I ran into the same thing you mentioned at standup -- this will pipe the upgrade message to the CWL and make it invalid CWL.

1. Is there a bug on that?
2. Or, you can add `--script` to avoid it being piped (but not sure if that's really the intent): `dockstore tool cwl --entry quay.io/collaboratory/dockstore-tool-bamstats:1.25-7 --script > Dockstore.cwl`

Reply:
https://github.com/dockstore/dockstore/issues/4048